src/App.css

```diff
@@ -307,20 +334,37 @@
   max-width: 85rem;
 }
 
+.comments button {
+  border: none;
+  background: transparent;
+  color: var(--blue0);
+}
+
+.comments button:hover {
+  border: 0.01rem solid var(--blue0);
+  background: transparent;
+}
```

Comment:
Good job working on the navlinks, they look pretty now. However, you have not worked on the alignment of the book container to start and end where the navigation starts and end. Also, the book container is still overflowing out of its parent container, consider resolving this to make the UI match the provided design.

Your design
<img width="902" alt="aa" src="https://github.com/Amazinggracee/Bookstore/assets/84930355/5c9b0f24-520f-4a8f-9d82-db9afe6fa4b5">


Furthermore, the widths for the input field needs to be worked on a little as pointed in the last reviewer's screenshot. The width for the book title is bigger than the others and as a result causes it parent container to take up the full width of the container where it lives. Consider resolving
<img width="856" alt="title" src="https://github.com/Amazinggracee/Bookstore/assets/84930355/62fb7a15-a83a-4a92-b779-7464481d2c5f">
 this and cross check your design with the template before asking for another review.

Template design

![237646829-62c179de-99eb-4355-a2fa-f229053154c6](https://github.com/Amazinggracee/Bookstore/assets/84930355/8136967a-d969-4246-97a1-e40beddec417)